source/layouts/components/_reference_table_of_contents.haml

```diff
@@ -1,4 +1,3 @@
-%h3 Contents
-
 %nav.sl-c-list-navigation-wrapper.sl-c-list-navigation-wrapper--collapsible
+  %h3 Contents
   = documentation_toc
```

Comment:
Style preference: do you mind adding a carriage return between the h3 and the documentation_toc line?